other/materials_designer/specific_examples/slab_strontium_titanate.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>The comment is cryptic, let's instead put:</p><p> </p><p><code>Rotate material to correctly identify multiple terminations. Without the rotation, the current implementation only finds a single termination (as of 2024-12).</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>